exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_definitions(-DEXOTICA_DEBUG_MODE)
 
```

Comment:
It's a ros standard directory structure that way. It should have been done when the package was created in the first place.